lib/beaker-pe/pe-client-tools/install_helper.rb

```diff
@@ -16,58 +14,85 @@ def install_pe_client_tools_on(hosts, opts = {})
-            when /osx/
-              release_path = "#{opts[:dev_builds_url]}/#{product}/#{ opts[:pe_client_tools_sha] }/artifacts/apple/#{version}/#{opts[:puppet_collection]}/#{arch}"
-              package_base = product.dup
-              package_base << "-#{opts[:pe_client_tools_version]}" if opts[:pe_client_tools_version]
-              package_name = package_base.dup
-              package_name << '-1' if opts[:pe_client_tools_version]
-              installer    = package_name + '-installer.pkg'
-              package_name << ".#{variant}#{version}.dmg" if opts[:pe_client_tools_version]
-            else
-              raise "install_puppet_agent_on() called for unsupported " +
-                "platform '#{host['platform']}' on '#{host.name}'"
-            end
-
-            if package_name
-              case host['platform']
-              when /windows/
+              when /win/
+                package_name = nil
+                release_path = "#{opts[:dev_builds_url]}/#{product}/#{ opts[:pe_client_tools_sha] }/artifacts/#{variant}"
+                package_name << "_#{opts[:pe_client_tools_version]}-1#{host['platform'].codename}_#{arch}.deb"
                 generic_install_msi_on(host, File.join(release_path, package_name), {}, {:debug => true})
-              else
+              when /osx/
+                release_path = "#{opts[:dev_builds_url]}/#{product}/#{ opts[:pe_client_tools_sha] }/artifacts/apple/#{version}/#{opts[:puppet_collection]}/#{arch}"
+                package_base = product.dup
+                package_base << "-#{opts[:pe_client_tools_version]}"
+                package_name = package_base.dup
+                package_name << '-1' if opts[:pe_client_tools_version]
+                installer    = package_name + '-installer.pkg'
+                package_name << ".#{variant}#{version}.dmg"
                 copy_dir_local = File.join('tmp', 'repo_configs')
                 fetch_http_file(release_path, package_name, copy_dir_local)
                 scp_to host, File.join(copy_dir_local, package_name), host.external_copy_base
-
-                if host['platform'] =~ /debian|ubuntu|cumulus|huaweios/
-                  on host, "dpkg -i #{package_name}"
-                elsif host['platform'] =~ /osx/
-                  host.generic_install_dmg(package_name, package_base, installer)
-                else
-                  host.install_package( product )
-                end
-              end
+                host.generic_install_dmg(package_name, package_base, installer)
+              else
+                install_dev_repos_on(product, host, opts[:pe_client_tools_sha], '/tmp/repo_configs')
+                install_package(host, 'pe-client-tools')
             end
           end
-          #}}}
         end
 
+        # Taken from puppet acceptance lib
+        # Install development repos
+        def install_dev_repos_on(package, host, sha, repo_configs_dir)
```

Comment:
Could we use the built-in Beaker method for this? https://github.com/puppetlabs/beaker/blob/master/lib/beaker/dsl/install_utils/foss_utils.rb#L922